src/components/Test.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react'
```

Comment:
Does it works if we change the import to be:
`import React from 'react'`?

Reply:
I got this error when I import like you stated - 
![Screen Shot 2022-09-13 at 1 36 55 PM](https://user-images.githubusercontent.com/102122005/189972217-d1ebbf1a-c827-4b95-9d06-9b8a8d07db6f.png)

However, digging a bit more, I found a solution where I don't need to import and use react-jsx in tsconfig. 
https://github.com/SoftwareBrothers/adminjs/issues/233

Reply:
I am getting a esmoduleinterop flag error when I import it. 
![esmoduleinterop error](https://user-images.githubusercontent.com/102122005/189972485-5042656b-08a9-4eea-8c15-eb51c26e7210.png)

I looked at ways to mitigate this and found a solution that seems to work.